sturdy/pool_registry/pool_registry.py

```diff
@@ -0,0 +1,68 @@
+POOL_REGISTRY = {
+    "Sturdy Redact Aggregator": {
+        "user_address": "0xcFB23D05f32eA0BE0dBb5078d189Cca89688945E",
+        "assets_and_pools": {
+            "pools": {
+                "0x0669091F451142b3228171aE6aD794cF98288124": {
+                    "pool_type": "STURDY_SILO",
+                    "contract_address": "0x0669091F451142b3228171aE6aD794cF98288124",
+                },
+                "0xFa68707be4b58FB9F10748E30e25A15113EdEE1D": {
+                    "pool_type": "STURDY_SILO",
+                    "contract_address": "0xFa68707be4b58FB9F10748E30e25A15113EdEE1D",
+                },
+            }
+        },
+    },
+    "Sturdy Crvusd Aggregator": {
+        "user_address": "0x73E4C11B670Ef9C025A030A20b72CB9150E54523",
+        "assets_and_pools": {
+            "pools": {
+                "0x6311fF24fb15310eD3d2180D3d0507A21a8e5227": {
+                    "pool_type": "STURDY_SILO",
+                    "contract_address": "0x6311fF24fb15310eD3d2180D3d0507A21a8e5227",
+                },
+                "0x200723063111f9f8f1d44c0F30afAdf0C0b1a04b": {
+                    "pool_type": "STURDY_SILO",
+                    "contract_address": "0x200723063111f9f8f1d44c0F30afAdf0C0b1a04b",
+                },
+                "0x26fe402A57D52c8a323bb6e09f06489C8216aC88": {
+                    "pool_type": "STURDY_SILO",
+                    "contract_address": "0x26fe402A57D52c8a323bb6e09f06489C8216aC88",
+                },
+                "0x8dDE9A50a91cc0a5DaBdc5d3931c1AF60408c84D": {
+                    "pool_type": "STURDY_SILO",
+                    "contract_address": "0x8dDE9A50a91cc0a5DaBdc5d3931c1AF60408c84D",
+                },
+            }
+        },
+    },
+    "Morpho USDC Vaults": {
+        "user_address": "0xFA60E843a52eff94901f43ac08232b59351192cc",
+        "total_assets": 1000000000000,
+        "assets_and_pools": {
+            "pools": {
+                "0xd63070114470f685b75B74D60EEc7c1113d33a3D": {
+                    "pool_type": "MORPHO",
+                    "pool_model_disc": "CHAIN",
```

Comment:
This can be removed

Reply:
i'm assuming you're referring to the the `pool_model_disc` parameter? if so then yes you're right it can be removed.